quinn-proto/src/connection/mod.rs

```diff
@@ -3167,6 +3167,10 @@ impl RttEstimator {
             .map_or(self.latest, |x| cmp::max(x, self.latest))
     }
 
+    pub fn max(&self) -> Duration {
+        self.get().max(self.latest)
+    }
+
```

Comment:
Is it worth abstracting this when it's used in only one place? Also, it's confusing that `rtt.max()` is not the maximum observed RTT, whereas `rtt.min` *is* the minimum observed RTT.

Reply:
I think you previously argued in favor of accessors for reasons of keeping fields private? I think the same applies here, plus there's actually calculation going on here, so we're not just abstracting over the field. I'm open to a better name; all I could come up with was something like `max_smoothed_or_latest()` or some such, which seemed wordy and still not that clear.

Reply:
Ah, right, since we're moving this into its own module it makes sense.

Maybe name it `conservative` and give it a decent doc comment? Still not self-explanatory, but at least it doesn't imply the wrong thing.

Reply:
Done.

(Actually I guess the argument for this which is strongest IMO is that providing abstractions local to the data makes it easier to prevent divergent use of the data, of the sort you fixed in #822.)